app/src/main/java/com/example/chooseyourownbeer/Beers.kt

```diff
@@ -0,0 +1,63 @@
+package com.example.chooseyourownbeer
+
+object Beers {
+    data class Beer(
```

Comment:
la data class la sposterei da qui, sarà utilizzata anche in futuro. potresti creare una cartella chiamata model e metterla lì.